.github/spellcheck/.spellcheck.yml

```diff
@@ -7,12 +7,24 @@ matrix:
     - .github/spellcheck/.wordlist.txt
     encoding: utf-8
   pipeline:
-  - pyspelling.filters.markdown:
+  - pyspelling.filters.markdown:  
   - pyspelling.filters.html:
       comments: false
       ignores:
       - code
       - pre
+      - script
+      - style
+      - a
+      - img
+      - href
```

Comment:
I know of "href" as an _attribute_ that appears in an `a` tag; I am not familiar with "href" as an HTML tag itself.